src/webportal/src/app/job-submission/models/data/mount-directories.js

```diff
@@ -329,8 +329,8 @@ export class MountDirectories {
             'azurefile://' + data.accountName + '/' + data.shareName;
           break;
         case 'azureBlob':
-          returnValue =
-            'azureblob://' + data.accountName + '/' + data.containerName;
+        case 'dshuttle':
+          returnValue = `Azure Blob (accountName: ${data.accountName} containerName: ${data.containerName})`;
```

Comment:
Here also need to change according to `ufsType`. Please change this?

Reply:
Done.